openstudiocore/src/gbxml/ReverseTranslator.cpp

```diff
@@ -86,7 +86,7 @@ namespace gbxml {
   }
 
   ReverseTranslator::ReverseTranslator()
-    : m_lengthMultiplier(1.0)
+    : m_nonBaseMultiplier(1.0), m_lengthMultiplier(1.0)
```

Comment:
@jasondegraw what is difference between m_nonBaseMultiplier and m_lengthMultiplier?

Reply:
It accounts for the fact that yards apparently aren't covered in the unit factory, we probably could do something special only for the ones that aren't and use the ones that are, but to be consistent I went this way. It probably doesn't really need to be kept around, but I did so just in case.